pages/system/dropdowns.js

```diff
@@ -92,70 +106,79 @@ const SELECT_MENU_MAP = {
   '10': 'Libra',
   '11': 'Scorpio',
   '12': 'Sagittarus',
-};`}
-        </System.CodeBlock>
-        <br />
-        <System.P>
-          Define the Dropdown value states and handle the state change the when a dropdown value is selected.
-        </System.P>
-        <br />
-        <System.CodeBlock>
-          {`state = {
-  one: '1',
-  two: '3',
-  three: '1',
-};
-
-_handleChange = (e) => {
-  this.setState({ [e.target.name]: e.target.value });
 };`}
         </System.CodeBlock>
         <br />
         <System.P>Declare the Dropdown component.</System.P>
         <br />
         <System.CodeBlock>
-          {`<System.SelectMenu
-  name="one"
-  value={this.state.one}
-  category="horoscope"
-  onChange={this._handleChange}
-  options={SELECT_MENU_OPTIONS}>
-  {SELECT_MENU_MAP[this.state.one]}
-</System.SelectMenu>
+          {`class ExampleOne extends React.Component {
+   state = { exampleOne: '1' }
+
+   _handleChange = (e) => this._handleChange(
```

Comment:
This is not correct, you need

`_handleChange = e => this.setState({ [e.target.name]: e.target.value });`